src/protos/mod.rs

```diff
@@ -31,6 +31,17 @@ pub mod coresdk {
     }
     pub mod workflow_activation {
         include!("coresdk.workflow_activation.rs");
+        pub fn create_evict_activation(task_token: Vec<u8>, run_id: String) -> WfActivation {
+            WfActivation {
+                task_token,
+                timestamp: None,
+                run_id,
+                jobs: vec![WfActivationJob::from(
+                    wf_activation_job::Variant::RemoveFromCache(true),
+                )],
+                from_pending: true,
```

Comment:
Why do we have `from_pending` in the protocol? It doesn't seem like it should interest lang, am I wrong?

Reply:
Not anymore it doesn't. I can clean this up now because of this change